src folder is missing after npm install

For some reason the src folder is missing after 'npm install vuefire', which causes:
`Module not found: Error: Cannot resolve module 'vuefire' ` in webpack

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "vuefire",
   "version": "1.0.0",
   "description": "Firebase bindings for Vue.js",
-  "main": "src/vuefire.js",
+  "main": "dist/vuefire.js",
   "files": [
     "dist/**",
     "LICENSE",

